uiowa/uiowa
# i2556 mentor view

Closes #2556 


# How to test


1. `blt ds —site=grad.uiowa.edu`
2. `blt frontend`
3. Navigate to http://grad.local.drupal.uiowa.edu/dei/srop/mentors and make sure there’s a teaser view of mentors grouped by their department
